include/zephyr/kernel.h

```diff
@@ -959,6 +959,49 @@ __syscall void k_thread_priority_set(k_tid_t thread, int prio);
  *
  */
 __syscall void k_thread_deadline_set(k_tid_t thread, int deadline);
+
+/**
+ * @brief Set deadline expiration time for scheduler
```

Comment:
the brief for the new API being introduced is the same as the existing one for 

```
@brief Set deadline expiration time for scheduler
```

It should be different and reflect the absolute part.